src/components/gltf-model-plus.js

```diff
@@ -361,6 +365,8 @@ AFRAME.registerComponent("gltf-model-plus", {
         (this.inflatedEl = inflateEntities(this.model, this.templates, true, this.data.modelToWorldScale))
       ) {
         this.el.appendChild(this.inflatedEl);
+        this.el.object3D.matrixNeedsUpdate = true; // Not totally sure why this is necessary
```

Comment:
/shrug on this one. some kind of weird condition where the GLTF model is loading in so the "first update matrix happens" thing doesn't work. i'm not going to worry about it for now